index.html

```diff
@@ -225,5 +225,27 @@ <h3>Skills</h3>
             </ul>
             </div>
         </section>
+        <section class="contact">
+            <div>
+            <h2>I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.
+            </h2>
+            <form action="https://formspree.io/f/{https://formspree.io/f/xeqdywdz}" method="post">
+                <input type="text" placeholder="Full name" maxlength="30">
+                <input type="text" placeholder="Email address" name="email">
```

Comment:
- [ ] This email type should be email. You can find more about it [here](https://www.w3schools.com/tags/att_input_type_email.asp)